src/support/err.c

```diff
@@ -289,8 +293,13 @@ __eventv_gen_msg(WT_SESSION_IMPL *session, char *buffer, size_t *buffer_len, boo
         WT_ERROR_APPEND(
           p, remain, "[%s][%s]", WT_VERBOSE_CATEGORY_STR(category), verbosity_level_tag);
 
-        WT_ERROR_APPEND(p, remain, ": ");
-        WT_ERROR_APPEND_AP(p, remain, fmt, ap);
+        /* Message. */
+        if (strlen(msg) > 0)
+            WT_ERROR_APPEND(p, remain, ": %s", msg);
+
+        /* Progress. */
+        if (progress != NULL)
```

Comment:
I'd personally prefer we just maintain the original message here, as we're doing in the JSON case. That being just the operation string (i.e. removing this append). As the message comes via the `progress_handler`, it should provide enough context. Additionally it might look a bit weird coming out of [mongodb](https://github.com/10gen/mongo/blob/5268175398a5b5122abc8c3047dce642fa59922e/src/mongo/db/storage/wiredtiger/wiredtiger_util.cpp#L607-L610)
```
        LOGV2(22431,
              "WiredTiger progress",
              "operation"_attr = redact(operation),
              "progress"_attr = progress);
```